app/templates/login.html

```diff
@@ -75,6 +75,12 @@
             {% endif %}
 
             <button class="button-save">Log in</button>
+
+            <header class="page-heading-smaller">
```

Comment:
`<header>` doesn't really fit here, we probably should be using something else both here and in the supplier app